src/networks.json

```diff
@@ -1848,5 +1848,21 @@
     },
     "start": 2560501,
     "logo": "ipfs://QmXqprM1TPafQFLirQ7dMPMgou1hoUGuCvHYSWU9g8ocv8"
+  },
+  "813": {
+    "key": "813",
+    "name": "Qitmeer Mainnet",
+    "shortName": "MEER",
+    "chainId": 813,
+    "network": "mainnet",
+    "multicall": "0x55034b2cF530ae3A8fC1e2e4523F58496796610F",
```

Comment:
@fiboswap https://evm.meerscan.io/address/0x55034b2cF530ae3A8fC1e2e4523F58496796610F/contracts#address-tabs

Can verify and publish multicall code here?

Reply:
Verified